src/zenml/integrations/skypilot/__init__.py

```diff
@@ -30,52 +30,52 @@
 SKYPILOT_GCP_ORCHESTRATOR_FLAVOR = "vm_gcp"
 SKYPILOT_AZURE_ORCHESTRATOR_FLAVOR = "vm_azure"
 
+class SkypilotGCPIntegration(Integration):
+    """Definition of Skypilot Integration for ZenML."""
 
-class SkypilotAWSIntegration(Integration):
-    """Definition of Skypilot AWS Integration for ZenML."""
-
-    NAME = SKYPILOT_AWS
-    REQUIREMENTS = ["skypilot[aws]"]
+    NAME = SKYPILOT_GCP
+    REQUIREMENTS = ["skypilot[aws,gcp,azure]<=0.4.1"]
```

Comment:
why do we need all the 3 clouds for one integration?

Reply:
This is a potential bug that we have a ticket for to investigate, it doesn't matter what cloud we use, zenml always pick up skypilot[aws] and install it in the docker image. So the workaround was to set all 3 of them.